_drafts/SC3-1-1-html-lang-valid.md

```diff
@@ -0,0 +1,43 @@
+---
+name: SC3-1-1-html-lang-valid
+
+description:
+This rule checks the `xml:lang` and/or the `lang` attribute, and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree: The tree that HTML is parsed into.
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root node of the page, if it is an `<html>` element with a `lang` and/or `xml:lang` attribute.
+
+### Expectation
+
+- The value of the `lang` or `xml:lang` attribute conforms to one of the values from the list - [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+- NOTE: If `lang` and `xml:lang` exists, then only `lang` is tested for conformance.
```

Comment:
These should be two paragraphs, not two lists. There needs to be an empty line between them. Also the note should start with `**Note**: `, not `NOTE:`.